AmazonChimeSDKDemo/AmazonChimeSDKDemo/data/RosterAttendee.swift

```diff
@@ -14,11 +14,13 @@ public class RosterAttendee {
     let attendeeName: String?
     var volume: VolumeLevel
     var signal: SignalStrength
+    var attendeeStatus: AttendeeStatus
 
-    init(attendeeId: String, attendeeName: String, volume: VolumeLevel, signal: SignalStrength) {
+    init(attendeeId: String, attendeeName: String, volume: VolumeLevel, signal: SignalStrength, attendeeStatus: AttendeeStatus = AttendeeStatus.joined) {
```

Comment:
For audio mode mono and stereo, do we need to change RosterAttendee constructor here? are we showing audio mode?

Reply:
Reverted the change. The attendee status should always be joined.